app/public/css/styles.css

```diff
@@ -1577,3 +1308,36 @@ td {
 .flex-align {
   display: flex;
 }
+
+/* NAVIGATION */
+
+.navbar {
+  padding: 32px 0;
+}
+
+.navbar-wrapper {
+  max-width: var(--width);
+  width: 100%;
+  margin: 0 auto;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.navbar-heading {
+  display: none;
+}
+
+.navbar-link {
```

Comment:
Please add `:hover` version of `.navbar-link`.

Reply:
Designer decision was that no hover effect should be applied to the links, specifically asked about this as it seemed strange to me at first, that's why there's no `:hover` declaration

Reply:
What's the reason for not having no `:hover` on links?